.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=false
```

Comment:
There were a few different issues with incompatible dependencies, and Speakeasy suggests that if it is used within a monorepo this should be set. 
ref: https://www.speakeasy.com/guides/sdks/typescript-monorepo-tips